msgid ""
msgstr "Content-Type: text/plain; charset=UTF-8"

#: TFORM1.CAPTION
msgid "Соединение pptp"
msgstr "Connection pptp"

#: TFORM1.MENUITEM1.CAPTION
msgid "Подключить"
msgstr ""

#: TFORM1.MENUITEM2.CAPTION
msgid "Отключить"
msgstr ""

#: unit1.message0
msgid "Внимание!"
msgstr "Attention!"

#: unit1.message10
msgid "Выход без ава#209�ии"
msgstr "Exit without problem"

#: unit1.message11
msgid "Выход п#209�и ава#209�ии"
msgstr "Exit with problem"

#: unit1.message3
msgid "Сначала сконфигу#209�и#209�уйте соединение: Меню->Утилиты->Системные(или Меню->Инте#209�нет)->vpnpptp(Наст#209�ойка соединения VPN PPTP/L2TP)."
msgstr "First configure connection: Menu->Tools->System(or Menu->Internet)->vpnpptp(Configure VPN PPTP)."

#: unit1.message4
msgid "No ethernet. Cетевой инте#209�фейс для VPN PPTP/L2TP недоступен. Если же он доступен, то установите \"Не конт#209�оли#209�овать state сетевого кабеля\" в Конфигу#209�ато#209�е."
msgstr "No ethernet. Network interface for VPN PPTP/L2TP is not avalible. If it present, select \"Not control network cable state\" in Configurator."

#: unit1.message5
msgid "No link. Сетевой кабель для VPN PPTP/L2TP неподключен."
msgstr "No link. Network cable for VPN PPTP/L2TP is not connect."

#: unit1.message6
msgid "Соединение"
msgstr "Connection"

#: unit1.message7
msgid "установлено"
msgstr "is up"

#: unit1.message8
msgid "отсутствует"
msgstr "not avalible"

#: unit1.message9
msgid "No link. Сетевой кабель для VPN PPTP/L2TP неподключен. А #209�еконнект не включен."
msgstr "No link. Network cable for VPN PPTP/L2TP not connect. And reconnect is not on."

#: unit1.message12
msgid "Устанавливается соединение "
msgstr "Connecting "

#: unit1.message1
msgid "Запуск этой п#209�ог#209�аммы возможен только под админист#209�ато#209�ом или с #209�аз#209�ешения админист#209�ато#209�а. Нажмите <ОК> для отказа от запуска."
msgstr "The start of this program is possible only as root or with root's permission. Press <OK> for exit."

#: unit1.message13
msgid "Получены ма#209�ш#209�уты через DHCP. "
msgstr "Get routes via DHCP. "

#: unit1.message14
msgid "Обна#209�ужен новый IP-ад#209�ес VPN-се#209�ве#209�а. Соединение пе#209�енаст#209�аивается... "
msgstr "Found a new IP-address VPN-server. Connection is reconfigured... "

#: unit1.message15
msgid "VPN-се#209�ве#209� не пингуется. "
msgstr "VPN-server does not ping. "

#: unit1.message16
msgid "Доступ в инте#209�нет отсутствует..."
msgstr "No Internet access available..."

#: unit1.message17
msgid "DNS-се#209�ве#209� до поднятия VPN не отвечает или неко#209�#209�ектный ад#209�ес VPN-се#209�ве#209�а. "
msgstr "DNS server before raise VPN is not responding or incorrect address VPN-server. "

#: unit1.message18
msgid "Шлюз локальной сети не пингуется. "
msgstr "The local network gate is not pinged. "

#: unit1.message19
msgid "П#209�ове#209�ено! Инте#209�нет #209�аботает!"
msgstr "Сhecked! The Internet is working!"

#: unit1.message20
msgid "Обна#209�ужено совпадение remote ip address с IP-ад#209�есом самого VPN-се#209�ве#209�а. Соединение было пе#209�енаст#209�оено."
msgstr "Found to match with the remote ip address IP-address of the VPN-server. The connection has been reconfigured."

#: unit1.message21
msgid "Ошибка получения ма#209�ш#209�утов через DHCP. "
msgstr "Failed to get routes via DHCP. "

#: unit1.message23
msgid "DNS1-се#209�ве#209� до поднятия VPN не пингуется. "
msgstr "DNS1-server before raise VPN does not ping. "

#: unit1.message24
msgid "DNS2-се#209�ве#209� до поднятия VPN не пингуется. "
msgstr "DNS2-server before raise VPN does not ping. "

#: unit1.message25
msgid "Вы можете в конфигу#209�ато#209�е VPN PPTP/L2TP выб#209�ать опцию #209�аз#209�ешения пользователям уп#209�авлять подключением."
msgstr "You can use the configuration tool VPN PPTP/L2TP choose to permit users to manage the connection."

#: unit1.message26
msgid "Вы также можете сконфигу#209�и#209�овать соединение из Цент#209�а Уп#209�авления->Сеть и инте#209�нет->Наст#209�ойка VPN-соединений->VPN PPTP/L2TP."
msgstr "You can also configure a connection from the Control Center->Network and Internet->Configuring VPN-connections->VPN PPTP/L2TP."

#: unit1.message22
msgid "Статус:"
msgstr "Status:"

#: unit1.message27
msgid "Заг#209�ужено:"
msgstr "Download:"

#: unit1.message28
msgid "Отдано:"
msgstr "Upload:"

#: unit1.message29
msgid "В#209�емя в сети:"
msgstr "Online Time:"

#: unit1.message30
msgid "ч."
msgstr "h."

#: unit1.message31
msgid "м."
msgstr "m."

#: unit1.message32
msgid "с."
msgstr "s."

#: unit1.message33
msgctxt "unit1.message33"
msgid "ОК"
msgstr "OK"

#: unit1.message34
msgctxt "unit1.message34"
msgid "Да"
msgstr "Yes"

#: unit1.message35
msgctxt "unit1.message35"
msgid "Нет"
msgstr "No"

#: unit1.message36
msgctxt "unit1.message36"
msgid "Отмена"
msgstr "Cancel"

#: unit1.message37
msgid "Минуточку..."
msgstr "Please, wait..."

#: unit1.message38
msgid "Запускается п#209�иложение ponoff..."
msgstr "Application ponoff runs..."

#: unit1.message39
msgid "Наблюдать"
msgstr "Watch"

#: unit1.message41
msgid "Обна#209�ужено активное соединение dsl. Отключите его командой ifdown dsl0. Нажмите <ОК> для отказа от запуска."
msgstr "Detect an active connection dsl. Disable it with the command ifdown dsl0. Press <OK> for exit."

#: unit1.message40
msgctxt "unit1.message40"
msgid "DNS2-се#209�ве#209� п#209�и поднятом VPN не пингуется. "
msgstr "DNS2-server at elevated VPN does not ping. "

#: unit1.message42
msgid "DNS1-се#209�ве#209� п#209�и поднятом VPN не пингуется. "
msgstr "DNS1-server at elevated VPN does not ping. "

#: unit1.message2
msgid "Не обна#209�ужено ни одного се#209�виса, способного уп#209�авлять сетью. Ко#209�#209�ектная #209�абота п#209�ог#209�аммы невозможна!"
msgstr "Did not find any service that can manage the network. Correct operation of the program is impossible!"

#: unit1.message43
msgid "Рекомендуется в#209�учную уменьшить MTU/MRU, так как используемое значение MTU ="
msgstr "It is recommended to manually reduce the MTU/MRU, as the value to use MTU ="

#: unit1.message44
msgid "байт слишком большое."
msgstr "bytes is too large."

#: unit1.message45
msgid "Разме#209� файла-лога /var/log/syslog больше 1 GiB."
msgstr "The size of the log /var/log/syslog is more than 1 GiB."

#: unit1.message46
msgid "Возможны п#209�облемы с установлением соединения..."
msgstr "Possible problems with establishing a connection..."

#: unit1.message47
msgid "Запускается се#209�вис xl2tpd..."
msgstr "Service xl2tpd is starting..."
